app/src/main/java/com/example/tung/moviedb_07/data/model/ProductionCompany.java

```diff
@@ -2,37 +2,32 @@
 
 import android.os.Parcel;
 import android.os.Parcelable;
-import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
-/**
- * Created by tung on 6/8/17.
- */
-
 public class ProductionCompany implements Parcelable {
 
     @SerializedName("name")
-    @Expose
     private String mName;
-
     @SerializedName("id")
-    @Expose
-    private Integer mId;
-
-    public static final Creator<ProductionCompany> CREATOR = new Creator<ProductionCompany>() {
-
-        @SuppressWarnings({ "unchecked" })
-        public ProductionCompany createFromParcel(Parcel in) {
-            ProductionCompany instance = new ProductionCompany();
-            instance.mName = ((String) in.readValue((String.class.getClassLoader())));
-            instance.mId = ((Integer) in.readValue((Integer.class.getClassLoader())));
-            return instance;
-        }
-
-        public ProductionCompany[] newArray(int size) {
-            return (new ProductionCompany[size]);
-        }
-    };
+    private int mId;
+
+    public final static Creator<ProductionCompany> CREATOR =
+            new Creator<ProductionCompany>() {
+
+                @SuppressWarnings({
+                        "unchecked"
+                })
+                public ProductionCompany createFromParcel(Parcel in) {
+                    ProductionCompany instance = new ProductionCompany();
+                    instance.mName = ((String) in.readValue((String.class.getClassLoader())));
+                    instance.mId = ((int) in.readValue((int.class.getClassLoader())));
+                    return instance;
```

Comment:
ERROR: 'method def' child have incorrect indentation level 20, expected level should be one of the following: 12, 16.